clifford/test/test_numba_extensions.py

```diff
@@ -172,3 +172,23 @@ def test_pickling():
     e1._numba_type_  # int
     (e1 * 1.0)._numba_type_  # float
     assert pickle.loads(pickle.dumps(lt)) is lt
+
+
+def test_A_order():
+    import numpy as np
+
+    @numba.njit
+    def mul_mv(mv):
+        return mv*e3
+
+    mva = layout.MultiVector(np.ones(layout.gaDims))
+    mva.value = mva.value[::-1]
+    assert not mva.value.flags.c_contiguous
+
+    mvc = layout.MultiVector(np.ones(layout.gaDims))
+
+    with pytest.raises(ValueError):
+        mul_mv(mva)
```

Comment:
CI fails in two ways here:
* On jit-less mode, this doesn't error at all (as we'd expect) - we should skip this part of the test if jitting is disabled
* When jitting:
   * Numba catches this `ValueError`
   * Tries to recompile in object mode (`jit` not `njit`)
   * Fails, and throws `TypingError`

  The quick fix is to adjust the exception type - but falling back to object mode sounds like a dangerous thing to do performance-wise.